site/content/en/docs/manual/basics/creating_an_annotation_task.md

```diff
@@ -1,260 +1,252 @@
+| Data source          | Description                                                                                                                                                                                                                                         |
+| -------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| My computer          | Use this option to select files from your laptop or PC. <br> To select file: <br>1. Click on the **Select files** field: <br>![Select files](/images/select_files.jpg). <br> 2. Select files to upload.                                             |
+| Connected file share | **Advanced option**. <br>Upload files from a local or cloud shared folder. <br>**Note**, that you need to mount a fileshare first. <br>For more information, see [Share path](/docs/administration/basics/installation/#share-path)                 |
+| Remote source        | **Advanced option**.<br>Enter a list of URLs (one per line) in the field. <br>If you want a `manifest.json` file, select the **Use cache**. <br> For more information, see [Dataset manifest.](/docs/manual/advanced/dataset_manifest/)             |
+| Cloud Storage        | **Advanced option**. <br>To upload files from cloud storage, type the cloud storage name, choose the manifest file, and select the required files. <br> For more information, see [Attach cloud storage](/docs/manual/basics/attach-cloud-storage/) |
 
-URL link of the repository optionally specifies the path to the repository for storage
-(`default: annotation / <dump_file_name> .zip`).
-The .zip and .xml file extension of annotation are supported.
-Field format: `URL [PATH]` example: `https://github.com/project/repos.git [1/2/3/4/annotation.xml]`
+<!--lint enable maximum-line-length-->
 
-Supported URL formats :
+### RAW
 
-- `https://github.com/project/repos[.git]`
-- `github.com/project/repos[.git]`
-- `git@github.com:project/repos[.git]`
+The **Raw** is a way of working with labels for an advanced user.
 
-After the task is created, the synchronization status is displayed on the task page.
+> **Note:** Be careful with changing the raw specification of an existing task/project.
+> Removing any "id" properties will lead to losing existing annotations.
+> **This property will be removed automatically from any text you insert to this field**.
 
-If you specify a dataset repository, when you create a task, you will see a message
-about the need to grant access with the ssh key.
-This is the key you need to [add to your github account](https://github.com/settings/keys).
-For other git systems, you can learn about adding an ssh key in their documentation.
+![](/images/image126.jpg)
 
-### Use LFS
+Raw presents label data in _.json_ format with an option of editing and copying labels as text.
```

Comment:
Probably it is good to specify the purpose of the feature. It is easy to copy labels from one independent task to another.